src/fights.ts

```diff
@@ -1600,21 +1607,58 @@ export function freeFights(): void {
   }
 }
 
-function nepQuest() {
+function setNepQuestChoicesAndPrepItems() {
+  let quest = get("_questPartyFairQuest");
   if (get("_questPartyFair") === "unstarted") {
-    visitUrl("adventure.php?snarfblat=528");
-    if (get("_questPartyFairQuest") === "food") {
-      runChoice(1);
-      setChoice(1324, 2);
-      setChoice(1326, 3);
-    } else if (get("_questPartyFairQuest") === "booze") {
-      runChoice(1);
-      setChoice(1324, 3);
-      setChoice(1327, 3);
+    visitUrl(toUrl($location`The Neverending Party`));
+    quest = get("_questPartyFairQuest");
+    if (["food", "booze"].includes(quest)) {
+      print("Gerald/ine quest!", "blue");
+    }
+    if (["food", "booze", "woots", "trash", "dj"].includes(quest)) {
+      runChoice(1); // Accept quest
     } else {
-      runChoice(2);
-      setChoice(1324, 5);
+      runChoice(2); // Decline quest
+    }
+  }
+
+  if (get("lastEncounter") === "A Room With a View... Of a Bed" && lastDecision() === 5) {
+    set("_garbo_nepUsedRedDress", true);
+  }
+  if (get("lastEncounter") === "Basement Urges" && lastDecision() === 4) {
+    set("_garbo_nepUsedElectronicsKit", true);
+  }
+  if (get("lastEncounter") === "Gone Kitchin'" && lastDecision() === 3) {
+    print("Found Geraldine!", "blue");
+  }
+  if (get("lastEncounter") === "Forward to the Back" && lastDecision() === 3) {
+    print("Found Gerald!", "blue");
+  }
+
+  if (quest === "food") {
```

Comment:
It looks like we never set 1324 back to 5 for gerald/ine quests--in the event that we get a noncombat early enough to get a second one, shouldn't we set it back to 5 once we've found our food/booze target